src/utils/index.js

```diff
@@ -52,3 +52,10 @@ export function unprefixKeys(obj, prefixToRemove, lowercaseFirst = true) {
     return prev;
   }, {});
 }
+
+/**
+ * Returns a className string containing all non-empty classes from the given list of candidates.
+ */
+export function joinClasses(...candidates) {
```

Comment:
I don't have an objection, it doesn't have any dependencies, and it is endorsed by React themselves so I'd like to see that.